scripts/scenario_08_atlantis.lua

```diff
@@ -906,9 +906,9 @@ function scrambleMessage(message)
         local pos = irandom(1, #message - 1)
         message = message:sub(0, pos) .. "$" .. message:sub(pos + 1)
     end
-    message = [[(The transmission is loaded with static noise.)
+    message = string.format(_([[(The transmission is loaded with static noise.)
 
-]] .. message
+]] %s), message)
```

Comment:
this looks off, I think it needs to be ` %s]]), message)

Reply:
Sorry, It's a mistake of mine